src/modules/interact.js

```diff
@@ -51,6 +51,7 @@ const clearCompletedTasks = () => {
   const clearButton = document.querySelector('.item-container__remove-all');
```

Comment:
- Great job so far 👍 your functionality is good but there are still a few issues you need to work on.
- [ ] When an item's checkbox is clicked, the completed key has to change to true but that is not the case. Kindly address this issue.
- [ ] Also, kindly update the tasks' index value so that if a user adds a second element, its index will be 2 and so on. Right now all your tasks index remain at 1. It would be great if you could fix this.
- Your site:
![Annotation 2022-03-26 045633](https://user-images.githubusercontent.com/85113567/160224073-e1c828ae-6822-4174-bd0f-2f42a3e7f19e.jpg)